src/doublyLinkedList.js

```diff
@@ -0,0 +1,138 @@
+import Node from './node'
+
+export default class DoublyLinkedList {
+  constructor() {
+    this.length = 0
+    this.head = null
+    this.tail = null
+  }
+
+  getHeadNode = () => {
+    let node = this.head
+    return node
```

Comment:
instead of returning creating and immediately returning a local variable you can just return this.head